### PR TITLE
Archived patients selectable in trends and calculate dropdowns

### PR DESCRIPTION
**Pull request by MichaelEHammond**
Added a filter to remove archived patients from the fetched records before they are added to the dropdown.

### CODE REVIEW

**Review by edong36 — APPROVED** [review]
Looks good!

**Review by morganbroome — APPROVED** [review]
Looks good

File: client/src/pages/trends.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ const Trends = () => {
                 return;
             }
             const records = await response.json();
-            setRecords(records);
+            const active_records = records.filter((record) => record.archived === false);
```
Comment (NamkhangNLe) [inline client/src/pages/trends.js:28]: Nice work!

**Review by NamkhangNLe — APPROVED** [review]
Works as intended

**Pull request merged**